software/network/src/main/resources/org/apache/brooklyn/entity/network/bind/named.conf

```diff
@@ -20,14 +20,15 @@
 // Generated by Brooklyn on ${.now?string.short}
 //
 options {
-    listen-on port ${entity.dnsPort} { 127.0.0.1; ${driver.address}; };
+    // listen-on default is all interfaces
+    // listen-on port ${entity.dnsPort} { 127.0.0.1; ${driver.address}; ${driver.subnetAddress}; };
```

Comment:
would prefer `listen-on port ${entity.dnsPort} { any; };` to be explicit, and allow changing port number, then. same for IPV6 as `listen-on-v6 port ${entity.dnsPort} { any; };`